packages/ui/components/ShareWindow.svelte

```diff
@@ -10,7 +10,7 @@
 
   export let sliders: Sliders;
 
-  let showShareWindow = true;
+  let showShareWindow = false;
```

Comment:
@yustarandomname I'm guessing you only disabled this because it was annoying during debugging, and it should be enabled again before merging?

Reply:
correct :)